rotkehlchen/api/v1/encoding.py

```diff
@@ -1117,6 +1117,8 @@ def make_staking_query(  # pylint: disable=no-self-use
             **_kwargs: Any,
     ) -> Dict[str, Any]:
         order_by_attribute = data['order_by_attribute'] if data['order_by_attribute'] is not None else 'timestamp'  # noqa: E501
+        if order_by_attribute == 'event_type':
```

Comment:
Ah wait I misunderstood when we spoke in discord. What I meant is that they give you a "type" argument and you map it to whatever you need. If they give you something they should not give you it's a BAD REQUEST error and should fail the call

Reply:
Yes the code behaves in that way. I remap the only possible different value (this one) and any other invalid param is treated as a bad request

Reply:
But that's not what I meant. The interface between frontend and backend is the api and it's a contract that should be respected.

If they give an argument they should not, then that should be an error and not quietly mapped to something else.